src/main/java/gr/cytech/chatreminderbot/rest/BotResource.java

```diff
@@ -42,50 +49,45 @@ public String handleReq(Request req) {
             logger.warn("Error from message:{}", req.getMessage().getText(), e);
             message = "Not even a clue what you just said";
         }
+
+        //case message is already build in json
+        if (message.startsWith("{\"actionResponse\":{\"type\":")) {
```

Comment:
this is less than ideal... and very prone to breaking
I propose to replace the returned string with a more meaningful response, containing any other needed data to send it